inf-clojure.el

```diff
@@ -126,8 +131,9 @@ The following commands are available:
 \\{inf-clojure-minor-mode-map}"
   :lighter "" :keymap inf-clojure-minor-mode-map
   (make-local-variable 'completion-at-point-functions)
-  (add-to-list 'completion-at-point-functions
-               #'inf-clojure-completion-at-point))
+  (when inf-clojure-minor-mode-completions
```

Comment:
This is not a good solution as it will require the users to set this **before** loading inf-clojure, which will certainly lead to confusion. Generally such config option should be consulted at runtime to avoid such problems. This can also be done much simpler by setting `completion-at-point-functions` to `nil` in the mode's hook.